lib/project_types/script/commands/create.rb

```diff
@@ -30,8 +30,8 @@ def call(args, _name)
       end
 
       def self.help
-        allowed_values = "{{cyan:discount}} and {{cyan:unit_limit_per_order}}"
-        ShopifyCli::Context.message('script.create.help', ShopifyCli::TOOL_NAME, allowed_values)
+        allowed_values = Script::Layers::Application::ExtensionPoints.types.map { |type| "{{cyan:#{type}}}" }
+        ShopifyCli::Context.message('script.create.help', ShopifyCli::TOOL_NAME, allowed_values.join(', '))
```

Comment:
Did we do it this way just so we don't have the `,` in `cyan`?  It's ok in this case, but just an FYI, if you ever feel the code's complexity doesn't justifies the UX design, we should question it with the designer.  But, here something this minor is perfectly ok.

Looking at the output, changing from "discount and unit-limit-per-order" to "discount, unit-limit-per-order, payment-filter" looks ok, but I don't think this is scalable once we have more EPs.  For the next preview, we could possibly have a LOT more EPs (hoping to be like 8).  Maybe this is a question for @PhilMcClelland and @abater1970 (is this Annette Bater?) to see the best way to tell the user of all the possible values.  I can think of the following:
1. Don't display the list here at all in error message, because we prompt for the EP, and thus the user already has a way of seeing all the valid EPs.
2. Tell them to refer to the file: `extension_points.yml` for all the valid EPs.

WDYT?  Any other better ways of displaying this?

Reply:
(Just to budge in after our meeting earlier) I think the best bet would be to just link to the documentation of available EP types, once that documentation becomes available. But since we have a small number of EPs right now and docs don't exist yet, maybe it's fine to keep Erik's implementation for now.

Reply:
Sure, let's just keep it as is, and @erik-shopify let's open another ticket to track the future work.  Thanks!